### PR TITLE
bugfix: remove unsupported algorithm in jwt plugin.

### PR DESCRIPTION
**Pull request by moonming**
support algorithm: https://github.com/SkyLothar/lua-resty-jwt/blob/master/lib/resty/jwt.lua#L505

### What this PR does / why we need it:

 fix https://github.com/apache/apisix/issues/2325
### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible?

### CODE REVIEW

File: doc/plugins/jwt-auth.md [hunk 1]
```diff
@@ -20,11 +20,14 @@
...
-- [**Test Plugin**](#test-plugin)
-- [**Disable Plugin**](#disable-plugin)
+- [Summary](#summary)
+  - [Name](#name)
+  - [Attributes](#attributes)
+  - [How To Enable](#how-to-enable)
+  - [Test Plugin](#test-plugin)
+      - [get the token in `jwt-auth` plugin:](#get-the-token-in-jwt-auth-plugin)
+      - [try request with token](#try-request-with-token)
+  - [Disable Plugin](#disable-plugin)
```
Comment (Yiyiyimu) [inline doc/plugins/jwt-auth.md:30]: Maybe we need to revert the auto-format

Comment (moonming) [reply doc/plugins/jwt-auth.md:30]: Ok i didn't notice

**Review by membphis — APPROVED** [review]
LGTM.

If we can use the same "summary" style for English and Chinese, then this PR will be perfect.

File: doc/plugins/jwt-auth.md (hunk 1 above)
Comment (Yiyiyimu) [reply doc/plugins/jwt-auth.md]: revert the change. fixed 😃

**Review by Yiyiyimu — APPROVED** [review]

**Pull request merged**